exotations/dynamics_solvers/exotica_cartpole_dynamics_solver/test/test_cartpole_diff.py

```diff
@@ -14,18 +14,19 @@ class TestCartpoleDiff(unittest.TestCase):
     def dynamics(state, u):
         # Analytical dynamics from underactuated robotics
         #   http://underactuated.mit.edu
-        x, theta, xdot, thetadot = state
+        # x, theta, xdot, thetadot = state
+        theta, x, thetadot, xdot = state
```

Comment:
Should this file also have been reverted?

Reply:
It should've, yes